src/sentry/tasks/relay.py

```diff
@@ -270,15 +270,21 @@ def schedule_invalidate_project_config(
         slight delay to increase the likelihood of deduplicating invalidations but you can
         tweak this, like e.g. the :func:`invalidate_all` task does.
     """
-    transaction.on_commit(
-        lambda: _schedule_invalidate_project_config(
-            trigger=trigger,
-            organization_id=organization_id,
-            project_id=project_id,
-            public_key=public_key,
-            countdown=countdown,
+    with sentry_sdk.start_span(
+        op="relay.projectconfig_cache.invalidation.schedule_after_db_transaction",
+    ):
```

Comment:
Is it possible to search for spans in sentry? If so, how would you find outliers that have a lot of these spans for a single transaction?

Reply:
It's currently not possible. Given the likeliness of this issue happening, and the fact that transactions in sentry represent API endpoints and tasks, the combination of the increased visibility this span provides plus statsd metrics should make identifying the problem easy enough not to use a transaction.